lib/kanta_web/router.ex

```diff
@@ -1,5 +1,6 @@
 defmodule KantaWeb.Router do
   use Phoenix.Router
+  @dialyzer {:no_return, {:call, 2}}
```

Comment:
Why? `no_return` appears when you have a bad typespec lower in the stack(start by fixing from the bottom).
If there's an actual bug in dialyzer add a comment here.

Reply:
There is a problem with the Phoenix package:
```
deps/phoenix/lib/phoenix/router.ex:2:no_return Function call/2 has no local return.
```
Unfortunately, I haven't found any reliable solution other than silencing this warning.

Reply:
Maybe it's related to: https://github.com/phoenixframework/phoenix/issues/2750
I think I had the same issue and it disappeared after updating Elixir & Phoenix deps for me.
It's fine to keep, just add a comment there about the issue.